frontend/src/v4/services/hotjar.ts

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Lets move this to v5. even though it uses the v4 config is a new feature and it should be there.